packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -103,14 +103,68 @@
                             </strong>
                             <span
                                 data-ng-if="flavor.locationCompatibility.isLocalZone"
-                                data-translate="pci_project_flavors_zone_localzone"
                                 class="oui-badge oui-badge_promotion"
-                            ></span>
+                            >
+                                <span
+                                    data-translate="pci_project_flavors_zone_localzone"
+                                ></span>
+                                <button
+                                    type="button"
+                                    class="oui-popover-button"
+                                    data-oui-popover
+                                    data-oui-popover-scope="$ctrl"
+                                    data-oui-popover-template="pci_project_flavors_zone_localzone.html"
+                                ></button>
+                                <script
+                                    type="text/ng-template"
+                                    id="pci_project_flavors_zone_localzone.html"
+                                >
+                                    <p
+                                        class="text-center"
+                                        data-translate="pci_project_flavors_zone_localzone_tooltip"
+                                    ></p>
+                                    <a
+                                        class="fa fa-external-link ml-2"
+                                        aria-hidden="true"
+                                        data-ng-href="{{$ctrl.localZoneUrl}}"
+                                        title="{{ 'pci_project_flavors_zone_tooltip_link' | translate }}"
+                                        target="_blank"
+                                        data-translate="pci_project_flavors_zone_tooltip_link"
+                                    ></a>
```

Comment:
**nitpick:** Link should be in the same line.
```suggestion
                                  <p class="text-center">
                                      <span
                                        data-translate="pci_project_flavors_zone_localzone_tooltip"
                                    ></span>
                                    <a
                                        class="fa fa-external-link ml-2"
                                        aria-hidden="true"
                                        data-ng-href="{{$ctrl.localZoneUrl}}"
                                        title="{{ 'pci_project_flavors_zone_tooltip_link' | translate }}"
                                        target="_blank"
                                        data-translate="pci_project_flavors_zone_tooltip_link"
                                    ></a>
                                 </p>
```

Reply:
In the mock ups link is given in new line.